cmd/start.go

```diff
@@ -331,7 +331,7 @@ func runMasterNodOnColdHot(ctx context.Context, config *configs.Config) error {
 	log.WithContext(ctx).Info("Finished checking pastel config!")
 
 	// If create master node using HOT/HOT wallet
-	if pastelid, err = getMasternodeConf(ctx, config); err != nil {
+	if /*pastelid*/_, err = getMasternodeConf(ctx, config); err != nil {
```

Comment:
should remove

Reply:
this and related to it commented out temporary - i didn't review the code in start.go yet